index.html

```diff
@@ -0,0 +1,301 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Design Teardown</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Spectral&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lora:wght@700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <nav class="navbar flex zero-border tier-2">
+      <p class="main-nav-logo tier-3">Main Logo</p>
+      <p class="nav-element tier-4">1</p>
+      <p class="nav-element tier-4">2</p>
+      <p class="nav-element tier-4">3</p>
+      <p class="nav-element tier-4">4</p>
+      <p class="nav-element tier-4">5</p>
+      <p class="nav-element tier-4">6</p>
+      <p class="nav-element tier-4">7</p>
+      <p class="search-nav tier-3">Search</p>
+    </nav>
+    <!-- <div class="test grid">
+      <div class="tier-1">One</div>
+      <div class="tier-2">Two</div>
+      <div class="tier-3">Three</div>
+      <div class="tier-4">Four</div>
+      <div class="tier-5">Five</div>
+    </div> -->
```

Comment:
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))